build.sbt

```diff
@@ -170,69 +178,82 @@ lazy val serviceDiscoveryAssemblySettings = Vector(
     s"${name.value}_${semanticVersioningMajor(scalaVersion.value)}-${version.value}.jar",
 
   packageBin in Compile := (assembly in Compile).value)
+*/
+
+lazy val temporaryDependenciesSettings = Vector(
+  libraryDependencies ++= Seq(
+    "com.typesafe.akka"        %% "akka-actor"          % Versions.akka              % "provided",
+    "ru.smslv.akka"            %% "akka-dns"            % Versions.akkaDns
```

Comment:
why need it after all?

Reply:
We do need it because the `ServiceLocator` and `LagomServiceLocator` will locate other service using DNS SRV lookup.

We don't need it for Akka cluster bootstrap though.

Reply:
> We don't need it for Akka cluster bootstrap though.

I see; The other service discovery you need specifically via SRV, why is that -- just for my understanding, because the ports also need to be discovered?

Reply:
That is correct, yes.

Reply:
We could use Kubernetes HTTP API for service discovery as well / instead, we're already doing that for akka-management. Marathon has the same functionality too.